howso/engine/trainee.py

```diff
@@ -1613,6 +1614,26 @@ def react(
                   not guaranteed to be a new case (that is, a case not found
                   in original dataset.)
 
+        goal_features_map : dict of dict
+            A mapping of feature name to the goals for the feature which
+            force reevaluation of local data in reacts to pull the predicted
+            action values toward achieving the specified value or goal as
+            defined by this map. Valid keys in the map are:
+			"goal": "min" or "max", will make a prediction while minimizing or
+                maximizing the value for the feature or
+			"value" : somevalue, will make a prediction while approaching the
+                specified value
+			note: nominal features only support 'value', 'goal' is ignored.
+				  for non-nominals, if both are provided, only 'goal' is considered.
+
+            .. code-block::
+                :caption: Example goal features map:
+
+                {
+                    "feature_a" : { "goal": "max" },
+                    "feature_b" : { "value": 99 }
+                }
```

Comment:
These docstrings need to be formatted better so it renders correctly in the docs.

```suggestion
            A mapping of feature name to the goals for the feature, which
            force reevaluation of local data in reacts to pull the predicted
            action values toward achieving the specified value or goal as
            defined by this map. Valid keys in the map are:

                - "goal": "min" or "max", will make a prediction while minimizing
                  or maximizing the value for the feature.
                - "value": somevalue, will make a prediction while approaching
                  the specified value.

            .. note::
                Nominal features only support "value", "goal" is ignored.
                For non-nominals, if both are provided, only "goal" is considered.

            .. code-block::
                :caption: Example goal features map:

                {
                    "feature_a" : { "goal": "max" },
                    "feature_b" : { "value": 99 }
                }
```

Reply:
Formatting still needs improvement